### ISSUE TITLE
text only data 是怎么加入预训练的呢，有这种类型的dataset代码吗

### ISSUE THREAD

**Issue by baoblei** [ByteDance-Seed/Bagel#210]
我按照vlm的dataset类去掉了sequence plan中的image list，但是在pretrain_unified_navit.py中当混合预训练时visual_gen是true，那么会没有相应的padding_images,该如何处理这种错误呢

 for curr_step, data in enumerate(train_loader, start=train_step):
        data = data.cuda(device).to_dict()
        data_indexes = data.pop('batch_data_indexes', None)
        ce_loss_weights = data.pop('ce_loss_weights', None)
        with torch.amp.autocast("cuda", enabled=True, dtype=torch.bfloat16):
            if training_args.visual_gen:
                with torch.no_grad():
                    data['padded_latent'] = vae_model.encode(data.pop('padded_images'))
            loss_dict = fsdp_model(**data)

**Comment by Andy1621** [comment]
在yaml中设置生成数据集为`is_mandatory: True`保证采样
